sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1144,6 +1144,31 @@ def wait_for_port_config_done(self, namespace):
             if key in ["PortConfigDone", "PortInitDone"]:
                 break
 
+    def need_lp_mode_for_dpdeinit(self, api, appl):
```

Comment:
@CharlieChenEC I don't think we can keep add vendor specific checks in a code that is generic and expects all modules to follow CMIS spec